source/fundamentals/crud/write-operations/delete.txt

```diff
@@ -0,0 +1,210 @@
+.. _csharp-delete-guide:
+
+=================
+Delete a Document
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to remove documents from your MongoDB
+collections using delete operations.
+
+Sample Data
+~~~~~~~~~~~
+
+The example in this guide use the ``restaurants`` collection
+from the ``sample_restaurants`` database. The documents in this
+collection are modeled by the following ``Restaurant`` class:
+
+.. literalinclude:: /includes/fundamentals/code-examples/crud/delete.cs
+   :language: csharp
+   :dedent:
+   :start-after: start-model
+   :end-before: end-model
+
+This collection is from the :atlas:`sample datasets </sample-data>` provided
+by Atlas. See the :ref:`<csharp-quickstart>` to learn how to create a free MongoDB cluster
+and load this sample data.
+
+Delete Operations
+-----------------
+
+Use **delete operations** to remove data from MongoDB. Delete operations
+use the ``DeleteOne()`` method, which deletes *the first document* that
+matches your query filter, and the ``DeleteMany()`` method, which
+deletes *all documents* that match the filter.
+
+Delete a Document
+~~~~~~~~~~~~~~~~~
+
+The following code shows how you can use the asynchronous
+``DeleteOneAsync()`` method or the synchronous ``DeleteOne()`` method to
+delete one document.
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: delete-one-async
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var result = await _restaurantsCollection.DeleteOneAsync(filter);
+
+   .. tab:: Synchronous
+      :tabid: delete-one-sync
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var result = _restaurantsCollection.DeleteOne(filter);
+
+Delete Multiple Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following code shows how you can use the asynchronous
+``DeleteManyAsync()`` method or the synchronous ``DeleteMany()`` method to
+delete all matched documents.
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: delete-many-async
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var result = await _restaurantsCollection.DeleteManyAsync(filter);
+
+   .. tab:: Synchronous
+      :tabid: delete-many-sync
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var result = _restaurantsCollection.DeleteMany(filter);
+
+Find runnable examples using these methods under :ref:`additional
+information <additional-info>`.
+
+Parameters
+~~~~~~~~~~
+
+The ``DeleteOne()`` and ``DeleteMany()`` methods expect you to pass a
+*query filter* specifying which documents to match. The query filter
+determines which records are selected for deletion based on the criteria
+in :manual:`the query filter document</core/document/#query-filter-documents>`.
```

Comment:
issue: The definition for query filter follows its use up above

Reply:
moved content to introduction